packages/core/src/Services/logout.js

```diff
@@ -20,6 +20,8 @@ const doLogout = response => {
     localStorage.removeItem('closed_toast_notifications');
     localStorage.removeItem('is_wallet_migration_modal_closed');
     localStorage.removeItem('active_wallet_loginid');
+    localStorage.removeItem('config.account1');
+    localStorage.removeItem('config.tokens');
```

Comment:
[R] removeItem('active_loginid') is missing

Reply:
This is already handled in the app itself.